samples/web/content/apprtc/gcm_register.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/python2.4
+#
+# Copyright 2015 Google Inc. All Rights Reserved.
+
+"""AppRTC GCM ID Registration
+
+This module implements user ID registration and verification.
+"""
+
+import constants
+import datetime
+import json
+import logging
+import webapp2
+
+from google.appengine.ext import ndb
+
+def get_ancestor_key():
+  return ndb.Key('GCMRecord', 'global')
+
+PARAM_USER_ID = 'userId'
```

Comment:
Should these be added in constants instead? (Since responses like CODE_SENT specific to gcm_register are also there)